apps/server/src/modules/room-member/service/room-member.service.ts

```diff
@@ -38,11 +38,7 @@ export class RoomMemberService {
 		return roomMember;
 	}
 
-	private static buildRoomMemberAuthorizable(
-		roomId: EntityId,
-		group: Group,
-		roleSet: RoleDto[]
-	): RoomMemberAuthorizable {
+	private buildRoomMemberAuthorizable(roomId: EntityId, group: Group, roleSet: RoleDto[]): RoomMemberAuthorizable {
```

Comment:
any reason to remove the static?
there is no internal state in this function needed to prevent static

Reply:
see above.